lib/govuk_navigation_helpers/taxonomy_sidebar.rb

```diff
@@ -9,7 +9,7 @@ def initialize(content_item)
 
     def sidebar
       {
-        items: taxons
+        items: [taxons, elsewhere_on_govuk, elsewhere_on_the_web].flatten
```

Comment:
@alecgibson I think it's just a matter of personal preference, no other reason. I tend not to use `+` and `-` on things other than numbers.